components/SectionsColumn.js

```diff
@@ -163,9 +163,34 @@ export const SectionsColumn = ({
 
   let alphabetizedSectionSlugs = sectionSlugs.sort()
 
+  const collapseSidebar = () => {
```

Comment:
I like this idea a lot! I feel like it would be more React-y to have a `isSidebarCollapsed` state that toggles, and conditionally set the classNames/CSS inside of the JSX based on that state. What do you think?

Reply:
Ok, i'm working on it...